### ISSUE TITLE
In editor, moving a node with relative children doesn't update imagemap

### ISSUE THREAD

**Issue by howardjones** [howardjones/network-weathermap#95]
When a node has children (relative positions), and it is moved, the positions of the children are not immediately recalculated.

This gives a situation where the imagemap doesn't update, but the image itself does (because it involves reloading the config). After a node move, relative positions should be recalculated.

Temporary fix: just press "do nothing" or edit something else.

**Comment by ikorzha** [comment]
I am aware of the bug in #95, I experienced many times in the past. to get around it I usually reload a config in editor by open a new config file:).   Fix for it would be nice!

**Comment by ikorzha** [comment]
I have tested the latest editor.php build, I do see that original #95 bug still there. I'm not sure if you have made an attempt to address it or now.

**Comment by howardjones** [comment]
No, that will take more change - it needs to reset the status of all the nodes and recalculate them.
